### ISSUE TITLE
Docfx failing to resolve references

### ISSUE THREAD

**Issue by jskeet** [GoogleCloudPlatform/google-cloud-dotnet#327]
Opening this here rather than in the docfx repo for the moment, in case it's a local problem.

Having upgraded to docfx 2.3.0.0 (and to dotnet cli, which may be relevant) we're getting a lot of warnings like this:

```text
Warning: Invalid cref value "!:StorageService" found in triple-slash-comments for StorageClient defined in src/Google.Storage.V1/StorageClient.UploadObject.cs Line 23, ignored.
Warning: Invalid cref value "!:Google.Apis.Storage.v1.Data.Object" found in triple-slash-comments for CopyObject defined in src/Google.Storage.V1/StorageClient.CopyObject.cs Line 35, ignored.
Warning: Invalid cref value "!:Google.Apis.Storage.v1.Data.Object" found in triple-slash-comments for CopyObjectAsync defined in src/Google.Storage.V1/StorageClient.CopyObject.cs Line 59, ignored.
Warning: Invalid cref value "!:GoogleCredential" found in triple-slash-comments for CreateAsync defined in src/Google.Storage.V1/StorageClient.cs Line 60, ignored.
Warning: Invalid cref value "!:GoogleCredential" found in triple-slash-comments for Create defined in src/Google.Storage.V1/StorageClient.cs Line 74, ignored.
```

Will try to reproduce in a little tiny example.

**Comment by jskeet** [comment]
Filed https://github.com/dotnet/docfx/issues/628

**Comment by jskeet** [comment]
Now fixed by using docfx 2.5.4.
